packages/reason-expo/src/Constants.re

```diff
@@ -32,20 +67,56 @@ type platformIOS = {
   systemVersion: string,
 };
 
-[@bs.deriving abstract]
+[@bs.deriving {abstract: light}]
+type platformAndroid = {versionCode: int};
+
+[@bs.deriving {abstract: light}]
 type platform = {
-  ios: platformIOS,
-  android: platformAndroid,
+   ios: platformIOS,
+   android: platformAndroid,
+ };
+[@bs.deriving {abstract: light}]
+type t = {
+  // "addListener": [Function anonymous],
+  appOwnership: string,
+  debugMode: bool,
+  deviceId: string,
+  deviceName: string,
+  deviceYearClass: int,
+  experienceUrl: string,
+  expoRuntimeVersion: string,
+  expoVersion: string,
+  getWebViewUserAgentAsync: unit => Js.Promise.t(string),
+  installationId: string,
+  isDetached: bool,
+  isDevice: bool,
+  isHeadless: bool,
+  linkingUri: string,
+  linkingUrl: string,
+  manifest: manifest,
+  nativeAppVersion: string,
+  nativeBuildVersion: string,
+  platform: platform,
+  // removeListeners: [Function anonymous],
+  sessionId: string,
+  statusBarHeight: int,
+  supportedExpoSdks: array(string),
+  systemFonts: array(string),
 };
 
-[@bs.module "expo-constants"] external platform: platform = "platform";
-
-[@bs.module "expo-constants"] external sessionId: string = "sessionId";
-
-[@bs.module "expo-constants"]
-external statusBarHeight: int = "statusBarHeight";
-
-[@bs.module "expo-constants"]
-external systemFonts: array(string) = "systemFonts";
-
-[@bs.module "expo-constants"] external manifest: Js.t({..}) = "manifest";
\ No newline at end of file
+[@bs.module "expo-constants"] external constants: t = "default";
+// let constants = constants;
```

Comment:
you can remove this comment, as well as all of the previous comments

Reply:
Done.